systemd/systemd.go

```diff
@@ -183,6 +186,14 @@ func Version() (int, error) {
 	return ver, nil
 }
 
+// AtLeast checks whether the installed version of systemd is greater or equal
+// than the given one. If systemd is not installed or not working, return
+// false.
+func AtLeast(requiredVersion int) bool {
+	version, err := Version()
+	return err == nil && version >= requiredVersion
```

Comment:
I think we should log the error somewhere like we did before this change, otherwise we won't know if something went wrong.

Reply:
I reowrked this a little, please have another look.